### PR TITLE
Add metadata operations saved metrics

### PR DESCRIPTION
**Pull request by yuzhu**

### CODE REVIEW

**Comment by yuzhu** [comment]
@calvinjia It was not clear to me which metadata operations we plan to account for. so I purposely did only a few to start, but the code is in place to expand the amount of metadata ops we track relatively easily. The overall idea is to use the actual operations done against alluxio - all the operations we did to ufs (during sync , load metadata etc)  and that difference should be the savings. Does that sound about right to you?

**Comment by yuzhu** [comment]
jenkins, test this please

**Comment by yuzhu** [comment]
alluxio-bot, test this please

**Review by calvinjia — CHANGES_REQUESTED** [review]
Can we also track the number of `listStatus` calls?

**Comment by yuzhu** [comment]
@calvinjia  updated to add listStatus and fine tuned the previous calculations.

**Comment by yuzhu** [comment]
image

**Comment by yuzhu** [comment]
@tieujason330 please review the front end part. @calvinjia PTAL the saved ufs operation metrics
Thanks

**Review by calvinjia — APPROVED** [review]

File: webui/master/src/containers/pages/Metrics/Metrics.tsx [hunk 1]
```diff
@@ -271,6 +271,22 @@ export class MetricsPresenter extends React.Component<AllProps> {
             </Table>
           </div>
         ))}
+
```
Comment (tieujason330) [inline webui/master/src/containers/pages/Metrics/Metrics.tsx:274]: you can remove this empty line

File: webui/master/src/containers/pages/Metrics/Metrics.tsx [hunk 2]
```diff
@@ -271,6 +271,22 @@ export class MetricsPresenter extends React.Component<AllProps> {
...
+        {Object.keys(data.ufsOpsSaved).map((key: string) => (
+          <div key={key} className="col-12">
+            <h5>Saved Under FileSystem Operations of {key}</h5>
+            <Table hover={true}>
+              <tbody>
```
Comment (tieujason330) [inline webui/master/src/containers/pages/Metrics/Metrics.tsx:279]: are table headers needed?

**Review by tieujason330 — COMMENTED** [review]
did you run `npm run test` and `npm run format` to see if any tests/formatting was changed?

File: webui/master/src/containers/pages/Metrics/Metrics.tsx (hunk 1 above)
Comment (yuzhu) [reply webui/master/src/containers/pages/Metrics/Metrics.tsx]: removed

File: webui/master/src/containers/pages/Metrics/Metrics.tsx (hunk 2 above)
Comment (yuzhu) [reply webui/master/src/containers/pages/Metrics/Metrics.tsx:278]: nope, to be consistent with above block

**Comment by yuzhu** [comment]
@tieujason330 ran npm tests and all passed. PTAL

**Review by tieujason330 — APPROVED** [review]

**Comment by yuzhu** [comment]
alluxio-bot, merge this please

**Comment by yuzhu** [comment]
alluxio-bot, cherry-pick this to branch-2.4 please

**Pull request merged**